data/src/main/java/com/linkedin/data/schema/annotation/AnnotationCheckResolvedPropertiesVisitor.java

```diff
@@ -40,9 +39,9 @@ public class AnnotationCheckResolvedPropertiesVisitor implements SchemaVisitor
 {
   private Map<PathSpec, Pair<CompatibilityCheckContext, Map<String, Object>>> _nodeToResolvedPropertiesMap = new HashMap<>();
 
-  private final String FIELD_INDICATOR = "$field";
-  private final String UNION_MEMBER_KEY_INDICATOR = "$unionMemberKey";
-  private final String TYPEREF_INDICATOR = "$typeref";
+  private static final String FIELD_INDICATOR = "$field";
+  private static final String UNION_MEMBER_KEY_INDICATOR = "$unionMemberKey";
+  private static final String TYPEREF_INDICATOR = "$typeref";
```

Comment:
can we remove this from this file since you moved to DataSchemaConstants.java?

Reply:
I can remove the typeref indicator. The other 2 are still needed...